tutorxqueue/templates/xqueue/build/xqueue/Dockerfile

```diff
@@ -7,7 +7,14 @@ RUN {% if is_buildkit_enabled() %}--mount=type=cache,target=/var/cache/apt,shari
     --mount=type=cache,target=/var/lib/apt,sharing=locked{% endif %} \
     apt update && \
     apt upgrade -y && \
-    apt install -y language-pack-en git python3 python3-pip python3-venv libmysqlclient-dev
+    apt install -y \
+        language-pack-en \
+        git \
+        python3 \
+        python3-pip \
+        python3-venv \
+        libmysqlclient-dev \
+        pkg-config
```

Comment:
What is exactly this package for?

Reply:
I assume the reason is the same as [this one](https://github.com/overhangio/tutor-notes/pull/30#discussion_r1399486890)

Reply:
There is a commit for pymysql https://github.com/openedx/xqueue/commit/fe8c57b03e68e209fbd286be98f489addac04382